context-free.md

```diff
@@ -1,22 +1,44 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+ContextFree can be used by people who want to create digital images, 
+especially those that involve repeating structures, through some simple
+programming.
 
 ## What is easy to do in this language? Why is it easy?
+It is quite easy to create images that involve many repeating shapes because 
+you can create multiples of many shapes and manipulate them using recursion.
 
-
-## What is hard to do in this language? Why is it hard?
-
+## What is hard to do in this language? Why is it hard? 
```

Comment:
Neither of us seem to have considered some of the more programming-related limitations of ContextFree when answering this question - it might have been interesting to include that. It would probably be hard to do a lot of stuff that would be simple in a general purpose language in ContextFree.